store/mongo/datastore_mongo.go

```diff
@@ -460,6 +460,58 @@ func (db *DataStoreMongo) UpdateDeviceGroup(ctx context.Context, devId model.Dev
 	} // to check the update count
 }
 
+func (db *DataStoreMongo) UpdateDevicesGroup(
+	ctx context.Context,
+	devIDs []model.DeviceID,
+	group model.GroupName,
+) (int64, int64, error) {
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	filter := bson.M{
+		"_id": bson.M{"$in": devIDs},
+	}
+	update := bson.M{
+		"$set": bson.M{
+			DbDevAttributesGroup: model.DeviceAttribute{
+				Scope: model.AttrScopeSystem,
+				Name:  DbDevGroup,
+				Value: group,
+			},
+		},
+	}
+	res, err := collDevs.UpdateMany(ctx, filter, update)
+	if err != nil {
+		return -1, -1, err
+	}
+	return res.MatchedCount, res.ModifiedCount, nil
+}
+
+func (db *DataStoreMongo) UnsetDevicesGroup(
+	ctx context.Context,
+	deviceIDs []model.DeviceID,
+	group model.GroupName,
+) (int64, error) {
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	filter := bson.D{
+		{Key: "_id", Value: bson.M{"$in": deviceIDs}},
+		{Key: DbDevAttributesGroupValue, Value: group},
+	}
+	update := bson.M{
+		"$unset": bson.M{
+			DbDevAttributesGroup: "",
```

Comment:
```go
DbDevAttributesGroup: 1,
```
right?

Reply:
It doesn't really matter what the value is, according to the docs:

> The specified value in the $unset expression (i.e. "") does not impact the operation.

https://docs.mongodb.com/manual/reference/operator/update/unset/